README.md

```diff
@@ -1,4 +1,6 @@
 ![](/images/icon-readme.png)
+
+Thanks to brianmichel, i just translate Juice to French people contact me for bug or other translate : contact@flayor.com
```

Comment:
Can we add a `Contributors` section to the end of the read me please? Also, I'm happy to acknowledge you for the work you've done on the project, but I do not want to have the contact email address as part of this projects README, but feel free to link to your github/twitter profile.